man/org.freedesktop.systemd1.xml

```diff
@@ -201,6 +201,7 @@ node /org/freedesktop/systemd1 {
                       out a(sss) changes);
       DisableUnitFiles(in  as files,
                        in  b runtime,
+                       out b carries_install_info,
```

Comment:
This is a compatibility break, so it cannot be done, do you have an alternative?

Reply:
Well, I know little about dbus...

Would adding another interface and make systemctl use it do?

Reply:
yes, in general that's the preferred approach

Reply:
On a second look, I don't think it breaks compatibility as originally `method_disable_unit_files_generic` returns `reply_install_changes_and_free(m, message, -1, changes, n_changes, error);`, so adding a new output doesn't harm?

Reply:
this is a public api, and the signature of dbus interfaces must remain fixed, otherwise external programs using this will break

Reply:
Thanks for the quick explanation. Not aware of the signature thing at all...